.github/workflows/linter.yml

```diff
@@ -17,15 +17,15 @@ jobs:
 
     steps:
     - name: Checkout
-      uses: actions/checkout@v2
+      uses: actions/checkout@v3
 
     - name: Use Node.js 12.x
-      uses: actions/setup-node@v1
+      uses: actions/setup-node@v3
       with:
         node-version: 12.x

```

Comment:
we should probably use `lts/*` here
And please, don't forget to also update the step name 😋